drivers/tee/optee/rpmb.c

```diff
@@ -131,8 +141,29 @@ static u32 rpmb_process_request(struct optee_private *priv, void *req,
 		mmc = get_mmc(priv, sreq->dev_id);
 		if (!mmc)
 			return TEE_ERROR_ITEM_NOT_FOUND;
-		if (mmc_rpmb_route_frames(mmc, RPMB_REQ_DATA(req),
-					  req_size - sizeof(struct rpmb_req),
+
+		rpmb_frame_data_sz = (req_size - sizeof(struct rpmb_req));
+		if (((unsigned long) RPMB_REQ_DATA(req)) & 0x03) // Unaligned
```

Comment:
no space between cast and casted value:
prefer remove `// some comment` at end of line.

`		if ((uintptr_t)RPMB_REQ_DATA(req) & 0x03)`